.gitignore

```diff
@@ -14,6 +14,7 @@ rm.**
 **/*pb*
 *.pb.go
 **pb**
+!**opba**
```

Comment:
**pb** may cause issues in the future, see if any pb files are unignored if you get rid of **pb**, if not then it's not needed as **/*pb* may do the job